pilot/pkg/networking/util/util.go

```diff
@@ -275,3 +277,22 @@ func ConvertLocality(locality string) *core.Locality {
 		}
 	}
 }
+
+// BuildConfigInfoMetadata builds core.Metadata struct containing the
+// name.namespace of the config, the type, etc. Used by Mixer client
+// to generate attributes for policy and telemetry.
+func BuildConfigInfoMetadata(config model.ConfigMeta) *core.Metadata {
+	return &core.Metadata{
+		FilterMetadata: map[string]*types.Struct{
+			istioMetadataKey: {
+				Fields: map[string]*types.Value{
+					config.Type: {
```

Comment:
I'd suggest using the same key here, and instead place the type into the value, a-la kubernetes style:
`/apis/GROUP/VERSION/namespaces/NAMESPACE/RESOURCETYPE`

Reply:
So, just for confirmation, you'd like to see something like:

```json
"filter_metadata": {
  "istio": {
     "destination-rule": "/apis/networking.istio.io/v1alpha3/namespaces/default/destinationrules/details-abort-dr"
  }
}
```

Is that universal?

Reply:
I don't know if it is universal, but it seems more robust. I meant something:
```yaml
filter_metadata:
  istio:
    config: networking.istio.io/v1alpha3/namespaces/default/destination-rule/details-abort-dr
```

Reply:
I'm concerned that we will create a UID that won't match what's in the API server (as `config.Type` doesn't necessarily match the RESOURCETYPE (`destination-rule` v. `destinationrules`, etc.). Thoughts?

Reply:
I think it'd be better to keep the same key `config` and use anything reasonable in URL. Seeing `route-rule` is surprising in the key.
We'll need to change proxy to extract the struct key, and using a consistent struct key seems like making that work easier.

Reply:
OK. Took your suggestion and updated. PTAL.